### PR TITLE
require `attr:` prefix when referencing attributes from span

### PR DESCRIPTION
**Pull request by mathnogueira**
This PR makes the prefix `attr:` required when referencing an attribute using expressions. **This requires frontend changes as well.**

## Checklist

- [x] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

File: server/encoding/yaml/conversion/parser/assertion_parser_test.go [hunk 1]
```diff
@@ -364,7 +364,6 @@ func TestParseAssertion(t *testing.T) {
 			require.NoError(t, err)
 			assert.Equal(t, testCase.ExpectedOutput.Attribute, output.Attribute)
 			assert.Equal(t, testCase.ExpectedOutput.Operator, output.Operator)
-			assert.Equal(t, testCase.ExpectedOutput.Value, output.Value)
```
Comment (mathnogueira) [inline server/encoding/yaml/conversion/parser/assertion_parser_test.go:367]: This was used when we had a `string` Value. Now, this is irrelevant and we should use assertExpression instead.

**Review by schoren — APPROVED** [review]
LGTM!

File: web/src/components/TestSpecForm/useExpectedInputLanguage.tsx [hunk 2]
```diff
@@ -25,7 +25,9 @@ export function useExpectedInputLanguage() {
             const identifierText = state.doc.sliceString(parentNode.from, parentNode.to);
             const isN = isNumber(identifierText);
 
-            const attributeOptions = Object.values(Attributes).map(s => ({label: s, apply: `${s} `}));
+            const attributeOptions = Object.values(Attributes)
+              .map(s => `attr:${s}`)
+              .map(s => ({label: s, apply: `${s} `}));
```
Comment (jorgeepc) [inline web/src/components/TestSpecForm/useExpectedInputLanguage.tsx:30]: Do we need the double map? 🤔 
```ts
.map(s => ({label: `attr:${s}`, apply: `attr:${s} `}));
```

Comment (cescoferraro) [reply web/src/components/TestSpecForm/useExpectedInputLanguage.tsx:30]: I dont think we need it. I just think it is more readable and dryer.
I can change if you want.

Comment (cescoferraro) [reply web/src/components/TestSpecForm/useExpectedInputLanguage.tsx]: Just committed the fix @jorgeepc

**Review by jorgeepc — APPROVED** [review]

**Pull request merged**